src/themes/teams/components/Input/inputVariables.ts

```diff
@@ -1,33 +1,42 @@
 import { pxToRem } from '../../../../lib'
 export interface InputVariables {
-  borderRadius: string
-  borderBottom: string
   backgroundColor: string
+  border: string
+  borderRadius: string
+  boxShadow: string
   fontColor: string
   fontSize: string
+  iconColor: string
   iconPosition: string
   iconRight: string
   inputPadding: string
-  inputFocusBorderColor: string
+  inputFocusBorderBottomColor: string
   inputFocusBorderRadius: string
 }
 
-const [_2px_asRem, _3px_asRem, _6px_asRem, _12px_asRem, _24px_asRem] = [2, 3, 6, 12, 24].map(v =>
-  pxToRem(v),
-)
+const [_1px_asRem, _2px_asRem, _3px_asRem, _6px_asRem, _12px_asRem, _24px_asRem] = [
+  1,
```

Comment:
this gets expanded during commit -- not sure that we want this level of prettiness?

Reply:
Let's delete this and just use `pxToRem(1)` for example.

>"Simple is better than complex."
>
> &mdash; https://en.wikipedia.org/wiki/Zen_of_Python